Legacy/bonej/src/main/java/org/bonej/plugins/ParticleCounter.java

```diff
@@ -2230,6 +2164,25 @@ private void joinStructures(final ImagePlus imp, final int[][] particleLabels,
 		return coordinates;
 	}
 
+	private static List<int[]> get6NeighbourhoodCoordinates(final short[] voxel,
+													 final int[] bounds)
+	{
+		final int x0 = voxel[0];
+		final int y0 = voxel[1];
+		final int z0 = voxel[2];
+		return get6NeighbourhoodCoordinates(x0, y0, z0, bounds[0], bounds[1], bounds[2]);
```

Comment:
why not `return get26NeighbourhoodCoordinates( voxels[0],voxels[1],voxels[2],bounds[0],...` ?